docs/data/material/components/autocomplete/Virtualize.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import TextField from '@mui/material/TextField';
 import Autocomplete, { autocompleteClasses } from '@mui/material/Autocomplete';
-import useMediaQuery from '@mui/material/useMediaQuery';
```

Comment:
The demo import should not change, please revert them.

Reply:
Reverted everything in docs/*